lib/name-tamer.rb

```diff
@@ -390,7 +395,9 @@ def name_case(lowercase)
   NONBREAKING_SPACE = "\u00a0"
   ASCII_SPACE       = "\u0020"
   ADFIX_JOINERS     = "[#{ASCII_SPACE}-]"
-  SLUG_DELIMITER    =  '-'
+  SLUG_DELIMITER    = '-'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.